source/protobuf/session.proto

```diff
@@ -12,6 +12,10 @@ service SessionUtilities {
   rpc EnumerateDevices(EnumerateDevicesRequest)
       returns (EnumerateDevicesResponse);
 
+  // Provides a list of software installed on server under localhost
+  rpc EnumerateSoftware(EnumerateSoftwareRequest)
```

Comment:
I would suggest calling it `EnumerateInstalledSoftware`. As we move forward with extending the API, there are cases where we want to consider available software that is not yet installed, and we have 'available' APIs for that.

Reply:
Updated, thanks for the suggestion!